system-test/restart-testcases/restart_test_automation.sh

```diff
@@ -21,20 +21,16 @@ if [[ "$ASYNC_NODE_INIT" = "true" ]]; then
   maybeAsyncNodeInit="--async-node-init"
 fi
 
-declare maybeExtraPrimordialStakes
-if [[ -n "$EXTRA_PRIMORDIAL_STAKES" ]]; then
-  maybeExtraPrimordialStakes="--extra-primordial-stakes $EXTRA_PRIMORDIAL_STAKES"
-fi
-
 # Restart the network
 "$REPO_ROOT"/net/net.sh stop
 
 sleep 2
 
 # shellcheck disable=SC2086
 "$REPO_ROOT"/net/net.sh start --skip-setup --no-snapshot-fetch --no-deploy \
-  --gpu-mode $startGpuMode $maybeAsyncNodeInit $maybeExtraPrimordialStakes
+  --gpu-mode $startGpuMode $maybeAsyncNodeInit
 
-# TODO add the test here
+# Basic test, check 5 transactions
+check_transaction_confirmations >> "$RESULT_FILE"
```

Comment:
This sanity test may not be resilient enough once we start testing restarts with a larger set of nodes.  I imagine it could take a network several minutes to sort out consensus if they've been running forked for several hours.

Instead reading the total active stake from `solana validators --output=json | jq .totalCurrentStake` might be a better way than running `solana ping`.  Basically poll `totalCurrentStake` once every 5-10 seconds, and then timeout after N seconds (not sure what N is, maybe 60 for now).